packages/core/src/components/dropdown/dropdown.tsx

```diff
@@ -528,18 +528,25 @@ export class TdsDropdown {
                   ref={(inputEl) => (this.inputElement = inputEl as HTMLInputElement)}
                   class={`${this.labelPosition === 'inside' ? 'placeholder' : ''}`}
                   type="text"
-                  placeholder={this.placeholder}
-                  value={this.getValue()}
+                  placeholder={this.filterFocus ? '' : this.placeholder}
+                  value={this.multiselect && this.filterFocus ? '' : this.getValue()}
                   disabled={this.disabled}
                   onInput={(event) => this.handleFilter(event)}
                   onBlur={(event) => {
```

Comment:
When putting filter to true and multiselect to false, choosing an option in the Dropdown and then blurring the input the chosen value is not displayed, only the placeholder. But when opening the Dropdown list again the value is marked as selected. Could it be a conditional rendering of the selected value that requires both filter and multiselect to be true?

Reply:
I'm not able to recreate this as seen in the recording
<img width="643" alt="dropdown" src="https://github.com/user-attachments/assets/6f921c73-b1bc-40d9-baef-5a80303508f2">

Reply:
Thank you for your explanation regarding this issue. Before proceeding with any further changes, I want to emphasize that the behavior you've described is currently what we have in production.

You can see it demonstrated here: https://tds-storybook.tegel.scania.com/?path=/story/components-dropdown--default&args=filter:true&globals=backgrounds.value:!hex(F9FAFB)

If this is not the desired behavior, I believe it would be better to address this in a new feature request rather than changing it here.

Reply:
On this one I agree with @ckrook , it is in prod too so lets create another ticket for it. I am afraid that adding new fixes under this one is out of scope and will make a review and test process more complex.